style.css

```diff
@@ -179,6 +177,154 @@ body {
   display: none;
 }
 
+.second {
+  background-color: #FAE8E4;
```

Comment:
Codacy found an issue: [Expected "#FAE8E4" to be "#fae8e4" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)